Automattic/wp-calypso
# Stepper: Remove a few unused steps and hooks

## Proposed Changes

While working on the React 18 upgrade in Calypso I spotted a few unused stepper steps and hooks.

This PR removes them.

## Testing Instructions

Verify the removed code is not in use, Calypso builds well and all checks are green.
